molecule/default/tests/test_default.py

```diff
@@ -4,40 +4,37 @@
 import os
 
 # Third-Party Libraries
-import pytest
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ["MOLECULE_INVENTORY_FILE"]
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("pkg", ["python3-pip", "python3-dev"])
-def test_python_debian(host, pkg):
+def test_pip(host):
     """Test that the appropriate packages were installed."""
-    if host.system_info.distribution == "debian" and host.system_info.release != "9.12":
-        assert host.package(pkg).is_installed
-
-
-@pytest.mark.parametrize(
-    "pkg", ["python-pip", "python-dev", "python3-pip", "python3-dev"]
-)
-def test_python_debian_9(host, pkg):
-    """Test that the appropriate packages were installed.
-
-    We treat Debian 9 as a special case because the CyHy AMIs (and
-    only the CyHy AMIs) are built on it.  Therefore it requires
-    python2.
-    """
-    if host.system_info.distribution == "debian" and host.system_info.release == "9.12":
-        assert host.package(pkg).is_installed
-
-
-@pytest.mark.parametrize("pkg", ["python3-pip", "python3-devel"])
-def test_python_redhat(host, pkg):
-    """Test that the appropriate packages were installed."""
-    if (
-        host.system_info.distribution == "fedora"
-        or host.system_info.distribution == "amzn"
-    ):
-        assert host.package(pkg).is_installed
+    amazon_extra_pkgs = ["python2-pip", "python-devel"]
+    debian_pkgs = ["python3-pip", "python3-dev"]
+    debian_stretch_extra_pkgs = ["python-pip", "python-dev"]
+    redhat_pkgs = ["python3-pip", "python3-devel"]
+    if host.system_info.distribution in ["debian", "kali", "ubuntu"]:
+        assert all([host.package(pkg).is_installed for pkg in debian_pkgs])
+        # We treat Debian 9 as a special case because the CyHy AMIs
+        # (and only the CyHy AMIs) are built on it.  Therefore it
+        # requires python2.
+        if (
+            host.system_info.distribution == "debian"
+            and host.system_info.codename == "9.12"
```

Comment:
Does `host.system_info.codename == "buster"` still not work? I believe it worked ok in https://github.com/cisagov/ansible-role-pip/pull/48, but if it's not working the latest Debian Stretch version is `9.13`.

Reply:
Please see commit 19d4e11.